packages/components/src/components/alert/component.tsx

```diff
@@ -26,6 +26,9 @@ const AlertIcon = (props: { label?: string; type?: AlertType }) => {
 	}
 };
 
+const buildLabel = (ariaLabel: string, label: string | undefined) => {
```

Comment:
Ich glaube da gabs ein Missverständnis: Das `label` sollte _nicht_ Teil des Icons werden, sonst wird es ja in jedem Fall zwei mal vorgelesen.
Es soll einfach nur (in jedem Fall) das `ariaLabel` als Icon label definiert werden.
Im A11y-Tree erscheint dann erst das Icon mit seinem Label, dann der Title.

Reply:
Dann hab ich es bisschen falsch verstanden. Ich nehme das Label dann raus aus dem Icon